<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PostgresMonitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icon</normaloff>:/icons/icon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widgetConnect" native="true">
       <property name="minimumSize">
        <size>
         <width>96</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="labelHost">
          <property name="text">
           <string>Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editHost">
          <property name="inputMask">
           <string>000.000.000.000</string>
          </property>
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPort">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editPort">
          <property name="inputMask">
           <string>000000</string>
          </property>
          <property name="text">
           <string>5432</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelUser">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editUser">
          <property name="text">
           <string>postgres</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPassword">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editPassword">
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>********</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>16</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>172</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionUpdate"/>
   <addaction name="actionAutoUpdate"/>
   <addaction name="actionPsql"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/connect</normaloff>:/icons/connect</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/update</normaloff>:/icons/update</iconset>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
  </action>
  <action name="actionPsql">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/psql</normaloff>:/icons/psql</iconset>
   </property>
   <property name="text">
    <string>psql</string>
   </property>
   <property name="toolTip">
    <string>psql</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/about</normaloff>:/icons/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionAutoUpdate">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/autoUpdate</normaloff>:/icons/autoUpdate</iconset>
   </property>
   <property name="text">
    <string>AutoUpdate</string>
   </property>
   <property name="toolTip">
    <string>AutoUpdate (1s)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
